examples/covid/ontario.case-details.yaml

```diff
@@ -31,8 +31,8 @@ content:
             "Row_ID" as id,
             to_timestamp(coalesce("Case_Reported_Date", "Test_Reported_Date")) as case_reported_date,
             case
-              when "Test_Reported_Date" = '' then null
-              else to_timestamp("Test_Reported_Date")
+              when "Test_Reported_Date" != '' then to_timestamp("Test_Reported_Date")
```

Comment:
This is a workaround for some obscure datafusion issue with types.

I spent an hour and couldn't find a clean case to submit an issue and it might be fixed in newer versions anyway (we are quite behind), so workaround should be fine for now.